res/controllers/Numark-Scratch-scripts.js

```diff
@@ -1,4 +1,4 @@
-const NumarkScratch = {};
+var NumarkScratch = {};
```

Comment:
this should prevent eslint messing this up.
```suggestion
// eslint-disable-next-line no-var
var NumarkScratch = {};
```